### PR TITLE
Add select selector for blueprints

### PR DESCRIPTION
**Pull request by EPMatt**
## Proposed change

Add documentation for the new select selector proposed here:
https://github.com/home-assistant/core/pull/45803
https://github.com/home-assistant/frontend/pull/8297

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [X] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: https://github.com/home-assistant/core/pull/45803
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: 

## Checklist

- [X] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [X] The documentation follows the Home Assistant documentation [standards][].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

File: source/_docs/blueprint/selectors.markdown [hunk 1]
```diff
@@ -381,6 +382,38 @@ This selector does not have any other options; therefore, it only has its key.
...
+
+The select selector shows a list of available options from which the user can choose. The value of the input contains the value of the selected option. Only a single option can be selected at a time.
+
+![Screenshot of a select selector](/images/blueprints/selector-select.png)
+
+The selector requires a list of options that the user can choose from.
+
+```yaml
+select:
+  options: []
```
Comment (thomasloven) [inline source/_docs/blueprint/selectors.markdown]: I don't think an empty list is a good thing to have in the documentation. Come to think of it, that should probably be disallowed in the backend code.

Comment (EPMatt) [reply source/_docs/blueprint/selectors.markdown]: Yeah, you're right. I thought `input_select` allowed to provide an empty list of options, but I checked and it's disallowed by the backend. I'll provide a fix in the backend PR and here.

Thank you for pointing it out. 👍🏻

**Comment by EPMatt** [comment]
I removed the reference to an empty list and used the example selector in the config snippet, so that the screenshot matches the provided yaml.

File: source/_docs/blueprint/selectors.markdown [hunk 2]
```diff
@@ -381,6 +382,29 @@ This selector does not have any other options; therefore, it only has its key.
...
+  options:
+    - Red
+    - Green
+    - Blue
+```
+
+{% configuration select %}
+options:
+  description: List of options that the user can choose from.
+  type: [list]
```
Comment (frenck) [inline source/_docs/blueprint/selectors.markdown]:
```suggestion
  type: list
```

**Review by frenck — CHANGES_REQUESTED** [review]

**Review by frenck — APPROVED** [review]
✅ Approved. Can be merged as soon as the parent PR gets merged.

**Pull request merged**